### PR TITLE
Add rmk

### PR DESCRIPTION
**Pull request by HaoboGu**
Add rmk: https://github.com/HaoboGu/rmk

I could only find the `embedded` section under `Development tools`, obviously rmk is not a development tool. So I added `embedded` to `Application`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,6 +269,10 @@ See also [Rust - Production](https://www.rust-lang.org/production) organizations
 * [vorot93/libmdbx-rs](https://github.com/vorot93/libmdbx-rs) [[mdbx-sys](https://crates.io/crates/mdbx-sys)] - Bindings for MDBX, a "fast, compact, powerful, embedded, transactional key-value database, with permissive license". This is a fork of mozilla/lmdb-rs with patches to make it work with libmdbx.
 * [WooriDB](https://github.com/naomijub/wooridb) - General purpose time serial database inspired by Crux and Datomic.
 
+### Embedded
+
+* [rmk](https://github.com/haobogu/rmk) - A feature-rich keyboard firmware.
```
Comment (palfrey) [inline README.md:274]: There's an extra space at the end of this line, which is what the build is complaining about.

**Comment by palfrey** [comment]
Nice project! I may at some point have a go rewriting https://tevps.net/blog/2023/05/30/lights-out-with-a-keyboard/ to use that, but sadly that keyboard uses an atmega32u4 which I'm seeing only limited embassy support for so far :(

File: README.md (hunk 1 above)
Comment (HaoboGu) [reply README.md]: Fixed

**Comment by HaoboGu** [comment]
yeah, atmega support is very limited in Rust.

so it's time to get a new keyboard :D

**Comment by HaoboGu** [comment]
oh sorry, it seems my VSC messed all the document.. I'm reverting the change

**Comment by HaoboGu** [comment]
@palfrey I think I've fixed everything, would you mind running the CI again?

**Pull request merged**